easy/Chalange1.js

```diff
@@ -0,0 +1,12 @@
+//Create a function that takes a number as an argument. Add up all the numbers from 1 to the number you passed to the function.
+
+function add(num)
+{
+let temp = num;
+while(num>0)
```

Comment:
Even if this logic works with `while` loop, it is bit time consuming to read and understand this logic.
I would prefer to do it using `for` loop which will be easy to understand.

Reply:
Done with the changes by my side. Please review once
commit ID 01